README.md

```diff
@@ -1,126 +1,248 @@
+We currently only support Ubuntu. If you aren't using Ubuntu, please visit [this page](./docs/00_install/01_build-from-source/00_build-unsupported-os.md) to explore your options.
 
-## Software Installation
-
-Visit the [release page](https://github.com/AntelopeIO/leap/releases) for Ubuntu binaries. This is the fastest way to get started with the software.
+You will need to know your version of Ubuntu to download the right binary. You can find this by going to Start/Windows/Super key > Settings > About > OS Name, or by running the following in your terminal.
+```bash
+lsb_release -a
+```
 
-### Building From Source
+<details> <summary>Ubuntu derivatives</summary>
 
-Recent Ubuntu LTS releases are the only Linux distributions that we fully support. Other Linux distros and other POSIX operating systems (such as macOS) are tended to on a best-effort basis and may not be full featured. Notable requirements to build are:
-* C++17 compiler and standard library
-* boost 1.67+
-* CMake 3.8+
-* (for Linux only) LLVM 7 - 11 (newer versions do not work)
+We currently only support Ubuntu but, if you want to try running our software on an Ubuntu derivative such as Linux Mint, you can determine the version of Ubuntu it was built from using the following command.
+```bash
+cat /etc/upstream-release/lsb-release
+```
 
-A few other common libraries are tools also required such as openssl 1.1+, libcurl, curl, libusb, GMP, Python 3, and zlib.
+</details>
 
-**A Warning On Parallel Compilation Jobs (`-j` flag)**: When building C/C++ software often the build is performed in parallel via a command such as `make -j $(nproc)` which uses the number of CPU cores as the number of compilation jobs to perform simultaneously. However, be aware that some compilation units (.cpp files) in Leap are extremely complex and will consume nearly 4GB of memory to compile. You may need to reduce the level of parallelization depending on the amount of memory on your build host. e.g. instead of `make -j $(nproc)` run `make -j2`. Failures due to memory exhaustion will typically but not always manifest as compiler crashes.
+Once you have a `*.deb` file downloaded for your version of Ubuntu, you can install it as follows.
+```bash
+sudo apt-get update
+sudo apt-get install -y ~/Downloads/leap*.deb
+```
+Your download path may vary.
+- If you are in an Ubuntu docker container, omit `sudo` because you run as `root` by default.
 
-Generally we recommend performing what we refer to as a "pinned build" which ensures the compiler and boost version remain the same between builds of different Leap versions (Leap requires these versions remain the same otherwise its state needs to be repopulated from a portable snapshot).
+Finally, verify Leap was installed correctly.
+```bash
+nodeos --full-version
+```
+You should see a [semantic version](https://semver.org) string print out followed by a `git` commit hash with no errors. For example:
```

Comment:
From [Merriam-Webster](https://www.merriam-webster.com/dictionary/printout), "print out" is a verb whereas "printout" is a noun. I was using it as a verb here, though either usage would make sense.

I just went ahead and removed "print" altogether because the sentence still makes sense without it and I don't want non-technical users to wonder if we might actually print something to their printer. As silly as it may sound, I've wondered that in the past reading documentation so it might be best to avoid the word "print" altogether.

Reply:
Ok I thought it was used a noun. No worries.